chimedb/dataflag/opinion/orm.py

```diff
@@ -0,0 +1,270 @@
+"""
+Table definitions for the CHIME data flag opinions.
```

Comment:
I would appreciate some high level documentation in this module (or elsewhere in the package) about what these concepts are and how they all interact, i.e. what's an opinion, what's a vote, how do they relate to, and how do they create flags? I think it would make it easier for others to understand exactly how this system works.